packages/unity-bootstrap-theme/webpack.config.js

```diff
@@ -30,7 +30,7 @@ const jsConfig = {
   devtool,
   entry: {
     "global-header": [path.resolve(paths.js, "global-header.js")],
-    "googleAnalytics": path.resolve(paths.js, "googleAnalytics.js"),
+    "global-header-datalayer": path.resolve(paths.js, "googleAnalytics.js"),
```

Comment:
The google analytics script is to be used for the whole package and not only for the header, maybe the name is not correct.

Reply:
I was looking to see where googleAnalytics was added https://github.com/ASU/asu-unity-stack/blob/dev/packages/bootstrap4-theme/gulpfile.js#L163-L166 But I think you are right that I got something wrong here,

Reply:
Agreed.

Reply:
Unresolving this.

Curious about the latest commit here: https://github.com/ASU/asu-unity-stack/pull/1034/commits/65443138f2d643e2ab3917a6da56e23621f8a205 We want to keep the global-header.js and googleAnalytics.js separate.  The concern that 
@maycomayco flagged at https://github.com/ASU/asu-unity-stack/pull/1034#pullrequestreview-1419504930 and which I seconded was the file naming - we should keep the file named googleAnalytics.js  and only containing the dataLayer code. Someone may be using the base theme but not the header, and they'll need the GA code.